web/themes/custom/recipe_theme/src/sass/view/_user_account_page.scss

```diff
@@ -0,0 +1,59 @@
+.user__name,
+.user__field-first-name {
+  .field__label,
+  .field__item {
+    display: flex;
+    justify-content: center;
+    font-family: 'Dosis', sans-serif;
+    font-weight: 300;
+    font-style: normal;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    color: $gray-800;
+  }
+}
+
+.user__name,
+.user__field-first-name {
+  .field__label {
```

Comment:
You don't need to move this nested rule in a separate block. You could add this right after line 13. Same goes for other similar rules.